lms/templates/darklang/preview_lang.html

```diff
@@ -11,5 +11,5 @@
 <%block name="bodyclass">pattern-library</%block>
 
 <%block name="content">
-    <%include file="/darklang/preview_lang_include.html" />
+    <%include file="/dark_lang/preview_lang_include.html" />
```

Comment:
Ah.. I missed finding that file in the PR.  I see it now.  Sorry for the false alarm.